osu.Game/Graphics/UserInterface/BackButton.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu-framework/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Transformations;
+using osu.Framework.Input;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    // Basic back button as it was on stable (kinda). No skinning possible for now
+    class BackButton : ExtendableButton
+    {
+        private TextAwesome icon;
+        private Vector2 iconPos = new Vector2(20, 0);
+
+        public BackButton()
+        {
+            InitialExtendLenght = new Vector2(40, 0);
+            ExtendLenght = new Vector2(60, 0);
+
+            RelativeSizeAxes = Axes.Y;
+            Width = 80;
+            //Height = 40; // should be set or should be relative?
+
+            Text = @"Back";
+
+            BGColour = new Color4(195, 40, 140, 255);
+            Colour = new Color4(238, 51, 153, 255);
+
+            Children = new Drawable[]
+            {
+                icon = new TextAwesome
+                {
+                    Anchor = Anchor.CentreLeft,
+                    TextSize = 25,
+                    Position = iconPos,
+                    Icon = FontAwesome.fa_osu_left_o
+                }
+            };
+
+            // HACK: because it never uses InitialExtendLenght that we give to it on creation
+            textContainer.Position = Position + InitialExtendLenght; 
+        }
+        protected override bool OnHover(InputState state)
+        {
+            bool result = base.OnHover(state);

```

Comment:
we don't care about the result of the base call here. we just want to return true if we are handling the hover (which we should be).